doc/source/whatsnew/v0.17.0.txt

```diff
@@ -101,56 +104,55 @@ Old                     New
 =====================   =============
 
 - Coercing types with ``DataFrame.convert_objects`` is now implemented using the
-keyword argument ``coerce=True``.  Previously types were coerced by setting a
-keyword argument to ``'coerce'`` instead of ``True``, as in ``convert_dates='coerce'``.
+  keyword argument ``coerce=True``.  Previously types were coerced by setting a
+  keyword argument to ``'coerce'`` instead of ``True``, as in ``convert_dates='coerce'``.
 
-  .. ipython:: python
+.. ipython:: python
```

Comment:
yes everything should be fully left-indendted (as you have a new section for this change)